tests/features/solution/add_solution.feature

```diff
@@ -35,7 +38,11 @@ Feature: "Add solution" visibility options.
     When I fill in the following:
       | Title             | Collection solution add solution |
       | Description       | This is a test text              |
-      | Documentation     | text.pdf                         |
+    And I attach the file "text.pdf" to "Documentation"
+    # Click the button to select an existing contact information.
+    And I press "Add existing"
+    And I fill in "Contact Information" with "Contact information example"
+    And I press "Add Contact information"
```

Comment:
This capitalization is not good, at least if it actually shows up in the interface like this. It should be "Add contact information".